apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -886,6 +907,27 @@ public static Coding createCoding(
     return new Coding(system, codeString, display);
   }
 
+  /**
+   * Creates a unit of measure coding.
+   *
+   * @param codingSystem the {@link Coding#getSystem()} to use
+   * @param codingCode the {@link Coding#getCode()} to use
+   * @return the output {@link Coding} for the specified input values
+   */
+  public static Coding createUnitOfMeasureCoding(String codingSystem, String codingCode) {
+    String display =
+        switch (codingCode) {
+          case "F2" -> TransformerConstants.CODING_SYSTEM_UCUM_F2_DISPLAY;
```

Comment:
Is it worth creating constants for these strings? I see this same listed is also used in `FissTransformerV2`

Reply:
Changed!